pkg/networking/middleware/response.go

```diff
@@ -40,11 +40,6 @@ func (rm ResponseMiddleware) RoundTrip(req *http.Request) (*http.Response, error
 
 	err = rm.errHandler(err, res.Request.Context())
 
-	// RoundTrip should return one or the other.
```

Comment:
Question: Are we sure that the response body is still consumable?

Reply:
At least my tests in CLI worked 🤷🏻‍♂️ Let's talk async.

Reply:
@PeterSchafer check out my latest commit. I didn't add the restored body, as I think we missed it's already happening: https://github.com/snyk/go-application-framework/blob/65a591adf20f92968937a652e43f42958647a308/pkg/networking/middleware/response.go#L83